model_compression_toolkit/core/pytorch/graph_substitutions/substitutions/reshape_with_static_shapes.py

```diff
@@ -63,6 +63,9 @@ def substitute(self,
         # configure the new static output shape attribute
         node.op_call_args = node.output_shape
 
+        # remove tensor indices that were converted to static op_calls
+        node.tensor_input_indices = node.tensor_input_indices[:sum([i != [1] for i in node.input_shape])]
```

Comment:
Try to split it a bit to make it more readable (and maybe expand the comment a bit).